src-ui/src/app/guards/dirty-form.guard.ts

```diff
@@ -0,0 +1,29 @@
+import { Injectable } from '@angular/core';
+import { DirtyCheckGuard } from '@ngneat/dirty-check-forms';
+import { Observable, Subject } from 'rxjs';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jonaswinkler/paperless-ng/pullRequest?prid=7592507)
```suggestion
import { Observable, Subject } from "rxjs";
```